src/Pager.js

```diff
@@ -96,6 +96,8 @@ const TIMING_CONFIG = {
 export default class Pager<T: Route> extends React.Component<Props<T>> {
   static defaultProps = {
     swipeVelocityThreshold: 1200,
+    extraSpringConfig: {},
```

Comment:
Let's rename these to `springConfig` and `timingConfig` and then make them updatable when you change props.